src/os/bluestore/BlueStore.cc

```diff
@@ -6909,8 +6909,11 @@ int BlueStore::_open_super_meta()
       ::decode(bluefs_extents, p);
     }
     catch (buffer::error& e) {
-      derr << __func__ << " unable to read bluefs_extents" << dendl;
-      return -EIO;
+      string kv_backend = cct->_conf->bluestore_kvbackend;
```

Comment:
That's because the other kv_backend except rocksdb can't use bluefs, which'if statement segment' from 3943 to 3946 says.
When i tried to use memdb, i was blocked at the catch clause that returns -EIO line 
memdb can't have key-value pair with bluefs_extents key...

Thanks


Thanks